src/util/debug.js

```diff
@@ -1,2 +1,3 @@
 // add artificial delay for testing
-export const sleep = (ms) => new Promise((resolve) => setTimeout(resolve, ms));
+export const sleep = (ms) =>
+  new Promise((resolve) => setTimeout(resolve, Math.round(ms)));
```

Comment:
As long as `ms` is a numerical value, you don't have to call `Math.round()` to convert it into an integer. (It doesn't hurt if you do.)